### ISSUE TITLE
qri validate should be possible before first save

### ISSUE THREAD

**Issue by hbruch** [qri-io/qri#1547]
I initialized a new dataset, which I want to validate before I commit it for the first time.

The respone I get is:

```
$ qri validate -b body.csv --structure structure.json

for linked dataset [mfdz/destatis_unfallorte]

repo: no history
```

**Comment by dustmop** [comment]
Good news! We just merged a solution for this a few days ago: https://github.com/qri-io/qri/pull/1541. Not sure if you're building from source, but if not, our next release will have this fixed. I'll leave this issue open until it arrives.

**Comment by b5** [comment]
should be fixed in 0.9.13. Thanks @hbruch!
